includes/Analytics.php

```diff
@@ -189,28 +189,6 @@ public function get_default_configuration( $tracking_id ) {
 						'send_to'        => $tracking_id,
 					],
 				],
-				// Fired when the bookend is shown to the user, after the last page of the current story.
```

Comment:
@swissspidy What should about these events?

Reply:
Looks like those triggers are now superfluous and can be safely removed.